python/simpler/worker.py

```
@@ -1529,12 +1534,26 @@ def handle_task() -> tuple[int, str]:
             # No-op when nothing is registered.
             if host_buf_ranges:
                 _rewrite_blob_host_addrs(buf, _OFF_TASK_ARGS_BLOB, host_buf_ranges)
+            pipeline_slot, pipeline_reserved, pipeline_generation = _PIPELINE_LEASE_FMT.unpack_from(
+                buf, _OFF_PIPELINE_LEASE
+            )
+            if pipeline_reserved != 0:
+                raise RuntimeError(f"chip_process dev={device_id}: invalid pipeline lease reserved field")
             # Hand the mailbox bytes straight to C++ (zero-copy zero-decode):
             # the blob layout is what `write_blob` already wrote, so re-parsing
             # it in Python is N×40B of avoidable work and a permanent
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)